addons/packages/fluxcd-helm-controller/0.17.2/README.md

```diff
@@ -0,0 +1,263 @@
+|-----------------|-------------------|------------------------------------------------------------------------------------|
+| `namespace`     | Optional          | Sets namespace for k8s objects where resources of helm-controller with be created. |
+| `limits_cpu`    | Optional          | Sets maximum usage of cpu for helm-controller deployment.                          |
+| `limits_memory` | Optional          | Sets maximum usage of memory for helm-controller deployment.                       |
+
+```yaml
+namespace: helm-system
+limits_cpu: 1000m
+limits_memory: 1Gi
+```
+
+## Installation
+
+To install FluxCD helm-controller from the Tanzu Application Platform package repository:
+
+1. Prerequisites:-
+
+   Install fluxcd-source-controller package:-
+
+   Helm controller package requires source-controller package to be installed for acquiring the Kubernetes manifests from the sources.
+
+    ```shell
+    tanzu package install fluxcd-source-controller -p fluxcd-source-controller.community.tanzu.vmware.com -v VERSION-NUMBER
+    ```
+
+   here `VERSION-NUMBER` is the version of the package
+
+   For example:
+
+    ```shell
+    tanzu package install fluxcd-source-controller -p fluxcd-source-controller.community.tanzu.vmware.com -v 0.21.2
+    \ Installing package 'fluxcd-source-controller.community.tanzu.vmware.com'
+    | Getting package metadata for 'fluxcd-source-controller.community.tanzu.vmware.com'
+    | Creating service account 'fluxcd-source-controller-default-sa'
+    | Creating cluster admin role 'fluxcd-source-controller-default-cluster-role'
+    | Creating cluster role binding 'fluxcd-source-controller-default-cluster-rolebinding'
+    | Creating package resource
+    / Waiting for 'PackageInstall' reconciliation for 'fluxcd-source-controller'
+    \ 'PackageInstall' resource install status: Reconciling
+
+    Added installed package 'fluxcd-source-controller'
+    ```
+
+2. List version information for the package by running:
+
+    ```shell
+    tanzu package available list helm-controller.fluxcd.community.tanzu.vmware.com
+    ```
+
+   For example:
+
+    ```shell
+    $ tanzu package available list helm-controller.fluxcd.community.tanzu.vmware.com
+    / Retrieving package versions for helm-controller.fluxcd.community.tanzu.vmware.com...  
+    NAME                                                    VERSION  RELEASED-AT  
+    helm-controller.fluxcd.community.tanzu.vmware.com        0.17.2   2022-02-23 16:44:08 +0530 IST  
+    ```
+
+3. Configure helm-controller package:
+
+   User can optionally provide the configuration parameters with --values-file flag while installing the package. Download the values.yaml file from [values.yaml](https://github.com/vmware-tanzu/package-for-helm-controller/blob/main/0.17.2/bundle/config/values.yaml).
+
+    ```shell
+    cat values.yaml  
+    #@data/values
+    ---
```

Comment:
```suggestion
```

The values.yaml should just contain the parameters that you're setting.

Reply:
removed unwanted lines